nntrainer/layers/acti_func.cpp

```diff
@@ -156,6 +156,11 @@ Tensor &ActiFunc::softmax(Tensor const &t, Tensor &output) {
   float *dp;
   float *rp;
 
+  /** TODO: fix this check */
```

Comment:
Could you elaborate on this comment? Is this enable strided softmax?

Reply:
Yes, it is to enable strided softmax in future. Updated the comment.